index.d.ts

```diff
@@ -0,0 +1,7 @@
+export declare function escapeId(val: string | string[], forbidQualified?: boolean): string;

```

Comment:
I think `val` should be of type `any` here, right?

Reply:
I am take another look at the possible paths.